compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1529,76 +1569,56 @@ private STNode parseFunctionTypeDescRhs(STNode metadata, STNode visibilityQualif
         return funcDef;
     }
 
-    private STNode parseVarDeclWithFunctionType(STNode metadata, STNode visibilityQualifier, List<STNode> qualifierList,
-                                                STNode functionKeyword, STNode funcSignature, boolean isObjectMember,
-                                                boolean isObjectTypeDesc) {
-
-        // --------------------------------------- Validate Qualifiers ---------------------------------------------
-
-        List<STNode> varDeclQualifiers = new ArrayList<>();
-        List<STNode> objectFieldQualifiers = new ArrayList<>();
-
+    private List<STNode> extractQualifiersOfFunctionType(List<STNode> qualifierList, boolean isObjectMember,
```

Comment:
may be "extractVarDeclOrObjectFieldQualifiers"? I think it reveals the true intention of the method.

Reply:
Done.